src/Prelude/Linear.hs

```diff
@@ -21,7 +21,7 @@ module Prelude.Linear
     -- * Unrestricted
     -- $ unrestricted
   , Unrestricted(..)
-  , unUnrestricted
+  , unLinear
```

Comment:
Do this in another PR, use `unUnrestricted` for now, and we change after. We should take the opportunity to bikeshed the name of the type and the data constructor, as well.